### ISSUE TITLE
Run-TestsInNavContainer fails when using NAV 2017 images

### ISSUE THREAD

**Issue by ernestasjuska** [microsoft/navcontainerhelper#527]
Hi,

Using Run-TestsInNavContainer on NAV 2017 containers shows this error:

```
docker : ClientSession is Uninitialized
At C:\Users\ernjus\Documents\WindowsPowerShell\Modules\navcontainerhelper\0.6.2.6\ContainerHandling\Invoke-ScriptInNavContainer.ps1:91 char:13
+             docker exec $containerName powershell $file
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (ClientSession is Uninitialized:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
```

NAV 2018 and BC containers work fine.

To reproduce:

```powershell
$imageName = "mcr.microsoft.com/dynamicsnav:2017-ltsc2019"
$containerName = "demo-test"
New-NavContainer -accept_outdated `
                 -accept_eula `
                 -imageName $imageName `
                 -containerName $containerName `
                 -updateHosts `
                 -includeTestToolkit `
                 -doNotExportObjectsToText
Run-TestsInBCContainer -containerName $containerName -credential $credential # shows error

$imageName = "mcr.microsoft.com/dynamicsnav:2018-rtm-ltsc2019"
$containerName = "demo-test"
New-NavContainer -accept_outdated `
                 -accept_eula `
                 -imageName $imageName `
                 -containerName $containerName `
                 -updateHosts `
                 -includeTestToolkit `
                 -doNotExportObjectsToText
Run-TestsInBCContainer -containerName $containerName -credential $credential # does not show error
```

Thanks.

PS. Full log (auth and license did not change anything)
```
NavContainerHelper is version 0.6.2.6
Host is Microsoft Windows 10 Enterprise - 1903
Docker Client Version is 18.09.2
Docker Server Version is 18.09.2
Using image mcr.microsoft.com/dynamicsnav:2017-ltsc2019
Removing container demo-test
Removing demo-test from hosts
Removing C:\ProgramData\NavContainerHelper\Extensions\demo-test
Creating Container demo-test
Using license file C:\Users\ernjus\Desktop\SB2017.flf
Version: 10.0.29930.0-W1
Generic Tag: 0.0.9.9
Container OS Version: 10.0.17763.557 (ltsc2019)
Host OS Version: 10.0.18362.239 (1903)
The container operating system does not match the host operating system, forcing hyperv isolation.
Using locale en-US
Using hyperv isolation
Disabling the standard eventlog dump to container log every 2 seconds (use -dumpEventLog to enable)
Files in C:\ProgramData\NavContainerHelper\Extensions\demo-test\my:
- AdditionalOutput.ps1
- AdditionalSetup.ps1
- license.flf
- MainLoop.ps1
- updatehosts.ps1
Creating container demo-test from image mcr.microsoft.com/dynamicsnav:2017-ltsc2019
b26703953ff4d33ff0dd34af19d01806716198cff22ba1d5cffadac89009471c
Waiting for container demo-test to be ready
Initializing...
Starting Container
Hostname is demo-test
PublicDnsName is demo-test
Using NavUserPassword Authentication
Starting Local SQL Server
Starting Internet Information Server
Creating Self Signed Certificate
Self Signed Certificate Thumbprint 07ED16C7DE15F60C1A3097E8D39E591A99876171
Modifying Service Tier Config File with Instance Specific Settings
Starting Service Tier
Creating Web Site
Creating Web Server Instance
Creating ManagementServicesPort and setting it to 7045
Using license file 'c:\run\my\license.flf'
Import License
Creating http download site
Setting SA Password and enabling SA
Creating admin as SQL User and add to sysadmin
Creating SUPER user
Container IP Address: 192.168.192.155
Container Hostname  : demo-test
Container Dns Name  : demo-test
Web Client          : http://demo-test/NAV/WebClient/

Files:

Initialization took 58 seconds
Ready for connections!
Reading CustomSettings.config from demo-test
Creating Desktop Shortcuts for demo-test
Importing Objects from C:\TestToolKit\CALTestCodeunits.W1.fob (container path)
Importing Objects from C:\TestToolKit\CALTestLibraries.W1.fob (container path)
Importing Objects from C:\TestToolKit\CALTestRunner.fob (container path)
TestToolkit successfully imported
Container demo-test successfully created
Importing Objects from C:\ProgramData\NavContainerHelper\Extensions\demo-test\PsTestTool-1\PSTestToolPage.fob (container path)
Objects successfully imported
docker : ClientSession is Uninitialized
At C:\Users\ernjus\Documents\WindowsPowerShell\Modules\navcontainerhelper\0.6.2.6\ContainerHandling\Invoke-ScriptInNavContainer.ps1:91 char:13
+             docker exec $containerName powershell $file
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (ClientSession is Uninitialized:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
```

**Comment by freddydk** [comment]
That is correct. I actually thought I had a version check in the code.
Not sure what it would take to support 2017 as well.

**Comment by ernestasjuska** [comment]
Thanks for quick answer. :)

I will try using alternative way to run tests (this looks promising https://community.dynamics.com/nav/b/richardrobbersenav/posts/running-automated-tests-from-powershell).

**Comment by freddydk** [comment]
Had a quick look - it should be possible to support 2017 as well with a few easy fixes.

**Comment by freddydk** [comment]
Both 2016 and 2017 will be supported from next version - and i have added tests to ensure that they don't stop working again.

**Comment by freddydk** [comment]
Shipped in 0.6.2.8
